docs/source/chainerx/install/index.rst

```diff
@@ -9,47 +9,81 @@ ChainerX, or ``chainerx``, can be installed as a top level Python package along
 
     Chainer must currently be installed from source in order to include ChainerX, but this is expected to change in the near future.
 
-Environment variables
----------------------
+Installing from source
+----------------------
 
-Configure the following environment variables before installing Chainer.
+The following environment variables are available for building ChainerX from source.
 
-========================== ================================================================================================
-Environment variable       Description
-========================== ================================================================================================
-``CHAINER_BUILD_CHAINERX`` ``1`` to build the ``chainerx`` package along with ``chainer``. ``0`` to skip. Default is ``0``.
-``CHAINERX_BUILD_CUDA``    ``1`` to build ``chainerx`` with CUDA support. ``0`` to skip. Default is ``0``.
-``CUDNN_ROOT_DIR``         Path to your cuDNN installation. Required when ``CHAINERX_BUILD_CUDA=1``.
-``CHAINERX_ENABLE_BLAS``   ``1`` to make BLAS enabled. ``0`` to disabled. Default is ``1``.
-========================== ================================================================================================
 
-Installing from source
-----------------------
+=========================== ========================================================================================================
+Environment variable        Description
+=========================== ========================================================================================================
+``CHAINER_BUILD_CHAINERX``  ``1`` to build the ``chainerx`` package along with ``chainer``. ``0`` to skip. Default is ``0``.
+``CHAINERX_BUILD_CUDA``     ``1`` to build ``chainerx`` with CUDA support. ``0`` to skip. Default is ``0``.
+                            See also :ref:`CUDA support <chainerx-install-cuda-support>` section below.
+``CHAINERX_ENABLE_BLAS``    ``1`` to make BLAS enabled. ``0`` to disabled. Default is ``1``.
+=========================== ========================================================================================================
+
 
 Simply run ``pip install --pre chainer`` after configuring the above environment variables.
+See :ref:`Examples <chainerx-install-cuda-support>` below.
+
+.. _chainerx-install-cuda-support:
+
+CUDA support
+------------
+
+When installing with the CUDA support, you also need to specify the cuDNN installation path since CUDA without cuDNN is currently not supported.
```

Comment:
```suggestion
When installing with the CUDA support, you also need to specify the cuDNN installation path since current implementation of ChainerX requires cuDNN.
```

Just that I don't like `not not` expression.

Reply:
There's virtually no possibility to support CUDA without cuDNN. I just removed the clause as I feel it's redundant. How do you think?

Reply:
I like short explanation.